### PR TITLE
Adding youtube channel link

### PR DESCRIPTION
**Pull request by AndrewBestbier**
## Gif
LGTM

## Motivation
[Trello Ticket](https://trello.com/c/iXUMQlUl)

### CODE REVIEW

File: assets/img/SVG/Youtube_green.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="23" height="17" viewBox="0 0 23 17" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><title>Youtube</title><defs><path id="a" d="M23 8.267v8.089H0V.179h23z"/></defs><g fill="none" fill-rule="evenodd"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path d="M9.126 11.25l-.001-6.462 6.214 3.242-6.213 3.22zM22.77 3.67s-.225-1.585-.915-2.284C20.982.47 20 .465 19.552.412 16.333.179 11.505.179 11.505.179h-.01s-4.828 0-8.047.233c-.45.053-1.43.058-2.304.973C.454 2.084.23 3.67.23 3.67S0 5.529 0 7.39v1.745c0 1.861.23 3.722.23 3.722s.224 1.585.914 2.284c.875.916 2.024.886 2.536.982 1.84.177 7.82.232 7.82.232s4.833-.008 8.052-.24c.45-.054 1.43-.058 2.304-.974.69-.699.915-2.284.915-2.284S23 10.998 23 9.136V7.391c0-1.861-.23-3.722-.23-3.722v.001z" fill="#CDE5B3" mask="url(#b)"/></g></svg>
```
Comment (zoeabryant) [inline assets/img/SVG/Youtube_green.svg]: Bear in mind the `_icons-svg.scss` created in #197 - we may need to revisit this and put it in that file

File: shared/components/ExternalPlatforms/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const ExternalPlatforms = () => (
+  <div className="ExternalPlatforms">
+    <a
+      target="_blank"
+      rel="noopener"
+      href={SLACK_URL}
+      className="ExternalPlatforms__btn ExternalPlatforms__btn--slack"
+    >Join us on Slack</a>
+    <a
```
Comment (zoeabryant) [inline shared/components/ExternalPlatforms/index.js]: Should these both use <ExternalLink instead of <a ?

Comment (lpil) [inline shared/components/ExternalPlatforms/index.js]: For sure!

File: assets/img/SVG/Youtube_green.svg (hunk 1 above)
Comment (lpil) [inline assets/img/SVG/Youtube_green.svg]: +1

**Pull request merged**